src/index.ts

```diff
@@ -6,7 +6,7 @@ import { intToHex, isHexString, stripHexPrefix } from 'ethjs-util';
 
 import { padWithZeroes } from './utils';
 
-export type TypedData = string | EIP712TypedData | EIP712TypedData[];
+export type TypedData = EIP712TypedData[];
```

Comment:
If `TypedData` is only used in / for V1, can we change its name to `V1TypedData` or something similar?

Reply:
We could :thinking: I had left it as-is to minimize breaking changes, but I guess we're past that mattering now.

Reply:
Updated! It's now called `TypedDataV1`